src/legacy/server/kbn_server.js

```diff
@@ -137,13 +130,6 @@ export default class KbnServer {
     const loggingConfig = config.get('logging');
     const opsConfig = config.get('ops');
 
-    const subset = {
-      ops: opsConfig,
-      logging: loggingConfig,
-    };
-    const plain = JSON.stringify(subset, null, 2);
-    this.server.log(['info', 'config'], 'New logging configuration:\n' + plain);
-
```

Comment:
These are already logged from the KP and having duplicates isn't necessary. We'll be removing the legacy logging system soon anyway but removing these sooner rather than later was a [direct request](https://github.com/elastic/kibana/issues/82431#issuecomment-794211967).